Mathlib/Algebra/Group/Subgroup/Basic.lean

```diff
@@ -1042,6 +1047,32 @@ theorem mem_sSup_of_directedOn {K : Set (Subgroup G)} (Kne : K.Nonempty) (hK : D
   simp only [sSup_eq_iSup', mem_iSup_of_directed hK.directed_val, SetCoe.exists, Subtype.coe_mk,
     exists_prop]
 
+-- non-pointwise lemma versions
```

Comment:
Why do you not use the pointwise operations? This seems like unnecessary code duplication

Reply:
Why do you care about being earlier than the `Pointwise` file? It's supposed to be a pretty basic file